lib/server-jobs.js

```diff
@@ -9,6 +9,9 @@ const logger = require('./logger');
 const socketServer = require('./socket-server');
 const sqldb = require('../prairielib/lib/sql-db');
 const sqlLoader = require('../prairielib/lib/sql-loader');
+const sqlDb = require('../prairielib/sql-db');
```

Comment:
Consistency with #6451
```suggestion
const sqldb = require('../prairielib/sql-db');
```